rebar.config

```diff
@@ -30,7 +30,8 @@
         {jsonrpc2, {git, "https://github.com/novalabsxyz/jsonrpc2-erlang",
                  {branch, "master"}}},
         {xorf, {git, "https://github.com/helium/erlang-xorf.git",
-                      {branch, "main"}}}
+                      {branch, "main"}}},
+        {backoff, {git, "https://github.com/ferd/backoff.git", {branch, "master"}}}
```

Comment:
backoff was already in the dep tree, why are you overriding it?

Reply:
because it wasnt a dep of miner, but of another dep ?

Reply:
its not a prob but i think its good practise to include a dep if its required by the app

Reply:
I guess I disagree, because it might break the behaviour of the library that originally pulled in the dep.

Reply:
for sake of haste i pulled the dep from miner